tensorrt_llm/_torch/pyexecutor/kv_cache_manager_v2.py

```
@@ -300,6 +301,35 @@ def _sync_host_tier_quota(host_quota: int, mapping: Mapping) -> int:
+            Distributed.get(mapping).allreduce(int(local_status), op=ReduceOp.MAX)
+        )
+    return local_status
+
+
+def _shutdown_kv_cache_manager_candidate(candidate: Optional[KVCacheManagerPy]) -> None:
+    """Best-effort cleanup for an uncommitted KV cache manager candidate."""
+    if candidate is None:
+        return
+    try:
+        candidate.shutdown()
+    except Exception as error:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)